webapp/src/pages/LoginPage/LoginPage.css

```diff
@@ -41,9 +40,9 @@
 }
 
 .login-form > .input,
-.button,
-.final-text,
-.login-container__separator {
+.login .button,
+.login .final-text,
+.login .login-container__separator {
```

Comment:
It seems that adding this `.login` makes button be wider than text in some cases:

https://user-images.githubusercontent.com/57688116/200768230-204156ce-cbe2-4744-a208-635d73e88bd1.mp4

And expected would be button to be width aligned with the text (main version):

https://user-images.githubusercontent.com/57688116/200768723-99038a23-316f-4d50-ba71-f30416b3ab16.mp4

Reply:
Okay, I see, I broke something else. Either way, that was conceptually wrong, because we were changing the style of the global button from the login page style sheet.
I was looking at where were we setting the width of the button to 15 in the components book page, and it was... in the login page css!